src/Illuminate/Database/Query/Grammars/PostgresGrammar.php

```diff
@@ -20,6 +21,43 @@ class PostgresGrammar extends Grammar
         'is distinct from', 'is not distinct from',
     ];
 
+    /**
+     * {@inheritdoc}
+     *
+     * @param Builder $query
```

Comment:
The PHPDoc of both methods is incorrect, take a look at `whereDate()`.

Reply:
Updated the docs, are they ok this way?